steps/operator/operator.go

```diff
@@ -0,0 +1,34 @@
+package olm
+
+import (
+	"github.com/getgauge-contrib/gauge-go/gauge"
+	"github.com/openshift-pipelines/release-tests/pkg/operator"
+	"github.com/openshift-pipelines/release-tests/pkg/store"
+	"github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
+)
+
+var _ = gauge.Step("Update TektonConfig CR to disbale RBAC resources", func() {
+	params := v1alpha1.Param{
+		Name:  "createRbacResource",
+		Value: "false",
+	}
+	operator.UpdateTektonConfigCR(store.Clients(), store.GetCRNames(), params)
```

Comment:
could we use `oc patch` command to update the config CR?

Reply:
updated